Pedia texts updates

MeuhV1 opened the pull request:
- Pedia texts for HR buildings
- Updated strategy texts for buildings and wonders
- Spanish Citadel provides culture like the vanilla Castle

dguenms commented on Assets/Python/CvRFCEventHandler.py:
```diff
@@ -441,7 +441,7 @@ def onCityGrowth(self, argsList):
 		if pCity.isHasRealBuilding(iEmpireStateBuilding):
 			iPop = pCity.getPopulation()
-			pCity.setBuildingCommerceChange(gc.getInfoTypeForString("BUILDINGCLASS_EMPIRE_STATE_BUILDING"), 0, iPop)
+			pCity.setBuildingCommerceChange(iEmpireStateBuilding, 0, iPop)
```
These won't work, `iEmpireStateBuilding` refers to the building, but the method requires the building class. The best we could do is `gc.getBuildingInfo(iEmpireStateBuilding).getBuildingClassType()`.

dguenms commented on Assets/Python/CvRFCEventHandler.py:
```diff
@@ -524,15 +524,15 @@ def onBuildingBuilt(self, argsList):
 		if iBuildingType == iEmpireStateBuilding:
 			iPop = city.getPopulation()
-			city.setBuildingCommerceChange(gc.getInfoTypeForString("BUILDINGCLASS_EMPIRE_STATE_BUILDING"), 0, iPop)
+			city.setBuildingCommerceChange(iEmpireStateBuilding, 0, iPop)
```
See above.

dguenms commented on Assets/Python/CvRFCEventHandler.py:
```diff
@@ -524,15 +524,15 @@ def onBuildingBuilt(self, argsList):
 				if city.getCityIndexPlot(i).isPeak():
 					iNumPeaks += 1
-			city.setBuildingCommerceChange(gc.getInfoTypeForString("BUILDINGCLASS_MACHU_PICCHU"), 0, iNumPeaks * 2)
+			city.setBuildingCommerceChange(iMachuPicchu, 0, iNumPeaks * 2)
```
See above.

dguenms commented on Assets/XML/Buildings/CIV4BuildingInfos.xml:
```diff
@@ -187,6 +187,16 @@
 			<iAsset>2</iAsset>
 			<iPower>3</iPower>
 			<fVisibilityPriority>1.0</fVisibilityPriority>
+			<ObsoleteSafeCommerceChanges>
```
👍

dguenms commented on Assets/XML/Text/National Wonders.xml:
```diff
@@ -215,28 +215,80 @@
 	</TEXT>
 	<TEXT>
+		<Tag>TXT_KEY_BUILDING_MUSEUM_STRATEGY</Tag>
+		<English>[COLOR_BUILDING_TEXT]Musea[COLOR_REVERT] greatle increase a city's cultural output. They also increase a city's [COLOR_HIGHLIGHT_TEXT]Great Person[COLOR_REVERT] birth rate depending on the culture level.</English>
```
greatly

dguenms commented on Assets/XML/Text/National Wonders.xml:
```diff
@@ -215,28 +215,80 @@
 	</TEXT>
 	<TEXT>
+		<Tag>TXT_KEY_BUILDING_ARMOURY_STRATEGY</Tag>
+		<English>[COLOR_BUILDING_TEXT]Armouries[COLOR_REVERT] increase a city's military unit production and provide culture.</English>
```
I think now that it's called Armoury instead of Military Academy, the culture bonus doesn't make much sense now.

dguenms commented on Assets/XML/Text/National Wonders.xml:
```diff
@@ -215,28 +215,80 @@
+	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_CENTRAL_BANK_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Central Bank[COLOR_REVERT] doubles its city's production of wealth and increases its chances of generating a [COLOR_UNIT_TEXT]Great Merchant[COLOR_REVERT].</English>
```
I would say "doubles its city's wealth output" for consistency, and to avoid confusion with the "build wealth" action.

dguenms commented on Assets/XML/Text/Buildings.xml:
```diff
@@ -316,16 +316,16 @@
-		<Tag>TXT_KEY_BUILDING_HARAPPAN_BATH_RESERVOIR</Tag>
-		<English>The [COLOR_BUILDING_TEXT]Reservoir[COLOR_REVERT], the Harappan replacement for the [COLOR_BUILDING_TEXT]Bath[COLOR_REVERT], doesn't require the [COLOR_TECH_TEXT]Medicine[COLOR_REVERT] technology, is cheaper and provides culture.</English>
+		<Tag>TXT_KEY_BUILDING_HARAPPAN_RESERVOIR_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Reservoir[COLOR_REVERT], the Harappan replacement for the [COLOR_BUILDING_TEXT]Bath[COLOR_REVERT], doesn't require the [COLOR_TECH_TEXT]Medicine[COLOR_REVERT] technology, does require the [COLOR_TECH_TEXT]Masonry[COLOR_REVERT] technology, is cheaper and does not provide culture.</English>
```
It's not intentional for Reservoirs and Baths to provide different culture.

dguenms commented on Assets/XML/Text/Buildings.xml:
```diff
@@ -316,16 +316,16 @@
 	<TEXT>
 		<Tag>TXT_KEY_BUILDING_HARBOR_STRATEGY</Tag>
-		<English>[COLOR_BUILDING_TEXT]Harbors[COLOR_REVERT] improve the food output of water plots. A [COLOR_BUILDING_TEXT]Harbor[COLOR_REVERT] is required to build [COLOR_BUILDING_TEXT]Customs Houses[COLOR_REVERT] and the [COLOR_BUILDING_TEXT]San Marco Basilica[COLOR_REVERT] and the [COLOR_BUILDING_TEXT]Trafalgar Square[COLOR_REVERT] world wonders.</English>
+		<English>[COLOR_BUILDING_TEXT]Harbors[COLOR_REVERT] improve the food output of water plots. A [COLOR_BUILDING_TEXT]Harbor[COLOR_REVERT] is required to build [COLOR_BUILDING_TEXT]Customs Houses[COLOR_REVERT], [COLOR_BUILDING_TEXT]Wharf[COLOR_REVERT] and the [COLOR_BUILDING_TEXT]San Marco Basilica[COLOR_REVERT] world wonder.</English>
```
Wharfs (plural as with Custom Houses).

dguenms commented on Assets/XML/Text/Buildings.xml:
```diff
@@ -374,7 +374,7 @@
 	<TEXT>
 		<Tag>TXT_KEY_BUILDING_INDONESIAN_CANDI_STRATEGY</Tag>
-		<English>The [COLOR_BUILDING_TEXT]Candi[COLOR_REVERT], the Indonesian replacement for the [COLOR_BUILDING_TEXT]Pagan Temple[COLOR_REVERT], provides one culture, one production and never goes obsolete.</English>
+		<English>The [COLOR_BUILDING_TEXT]Candi[COLOR_REVERT], the Indonesian replacement for the [COLOR_BUILDING_TEXT]Pagan Temple[COLOR_REVERT], provides one production and never goes obsolete.</English>
```
Should be a Monument replacement now, is it not?

dguenms commented on Assets/XML/Text/Buildings.xml:
```diff
@@ -386,15 +386,15 @@
 	<TEXT>
 		<Tag>TXT_KEY_BUILDING_JAIL_STRATEGY</Tag>
-		<English>[COLOR_BUILDING_TEXT]Jails[COLOR_REVERT] lower a city's [COLOR_HIGHLIGHT_TEXT]War Weariness[COLOR_REVERT] and increase espionage significantly.</English>
+		<English>[COLOR_BUILDING_TEXT]Jails[COLOR_REVERT] lower city maintenance costs and increase espionage. They are required to build [COLOR_BUILDING_TEXT]Constabula[COLOR_REVERT].</English>
```
Constabularies

dguenms commented on Assets/XML/Text/Buildings.xml:
```diff
@@ -386,15 +386,15 @@
 	<TEXT>
 		<Tag>TXT_KEY_BUILDING_NUCLEAR_PLANT_STRATEGY</Tag>
-		<English>[COLOR_BUILDING_TEXT]Nuclear Plants[COLOR_REVERT] provide pollution-free power for [COLOR_BUILDING_TEXT]Factories[COLOR_REVERT] with access to [COLOR_HIGHLIGHT_TEXT]Uranium[COLOR_REVERT].</English>
+		<English>[COLOR_BUILDING_TEXT]Nuclear Plants[COLOR_REVERT] provide pollution-free power with access to [COLOR_HIGHLIGHT_TEXT]Uranium[COLOR_REVERT].</English>
```
You should also mention the scientist slot and the ability to build ICBMs.

dguenms commented on Assets/XML/Text/Buildings.xml:
```diff
@@ -442,27 +442,27 @@
 	<TEXT>
 		<Tag>TXT_KEY_BUILDING_LABORATORY_STRATEGY</Tag>
-		<English>[COLOR_BUILDING_TEXT]Laboratories[COLOR_REVERT] increase a city's scientific research and its spaceship production, but also add unhealthiness.</English>
+		<English>[COLOR_BUILDING_TEXT]Laboratories[COLOR_REVERT] increase a city's scientific research and its spaceship production.</English>
```
It should be research with power, is it not?

dguenms commented on Assets/XML/Text/Buildings.xml:
```diff
@@ -883,16 +905,22 @@
 		<Tag>TXT_KEY_BUILDING_PROTESTANT_MONASTERY</Tag>
-		<English>Protestant Seminary</English>
-		<German>Protestantisches Seminar</German>
+		<English>Protestant Monastery</English>
```
It's still supposed to be a Seminary ...

dguenms commented on Assets/XML/Text/Buildings.xml:
```diff
@@ -918,16 +949,22 @@
 	<TEXT>
 		<Tag>TXT_KEY_BUILDING_ISLAMIC_TEMPLE</Tag>
 		<English>Islamic Mosque</English>
+		<French>Mosqu&#233;e</French>
```
Looks like there is an adjective missing here.

dguenms commented on Assets/XML/Text/Buildings.xml:
```diff
@@ -1176,38 +1272,78 @@
 	</TEXT>
 	<TEXT>
+		<Tag>TXT_KEY_BUILDING_ESTATE_STRATEGY</Tag>
+		<English>[COLOR_BUILDING_TEXT]Estates Towers[COLOR_REVERT] greatly increase a city's cultural output.</English>
```
Estates. Also no mention of artist/statesman slots?

dguenms commented on Assets/XML/Text/Buildings.xml:
```diff
@@ -1176,38 +1272,78 @@
+	<TEXT>
 		<Tag>TXT_KEY_BUILDING_STAR_FORT</Tag>
 		<English>Star Fort</English>
+		<French>Tracé à l'Italienne</French>
```
Looks like a unicode error here.

dguenms commented on Assets/XML/Text/Buildings.xml:
```diff
@@ -1176,38 +1272,78 @@
 	</TEXT>
 	<TEXT>
+		<Tag>TXT_KEY_BUILDING_ELECTRICAL_GRID_STRATEGY</Tag>
+		<English>[COLOR_BUILDING_TEXT]Electrical Grids[COLOR_REVERT] increase commerce output in cities with with access to power.</English>
```
Double "with".

MeuhV1 commented:
Updated comments.

Removed culture from Armoury. Added culture to the Harappan Reservoir so it matches the Bath.

dguenms commented on Assets/XML/Text/Buildings.xml:
```diff
@@ -936,7 +973,10 @@
 		<English>Islamic Grand Mosque</English>
-		<German>Islamische Gro�e Moschee</German>
+		<German>Islamische Gro&#223;e Moschee</German>
+		<German>Islamische Gro&#223;moschee</German>
```
Now there is German twice.

dguenms commented on Assets/XML/Text/Buildings.xml:
```diff
@@ -1176,38 +1272,78 @@
 	</TEXT>
 	<TEXT>
+		<Tag>TXT_KEY_BUILDING_DEPARTMENT_STORE_STRATEGY</Tag>
+		<English>[COLOR_BUILDING_TEXT]Department Stores[COLOR_REVERT] increase wealth in cities with with access to power.</English>
```
"with with"

dguenms reviewed (changes requested):
Only two more.

Commit:
Pedia texts for HR buildings
Updated strategy texts for buildings and wonders
Spanish Citadel provides culture like the vanilla Castle
Harappan Reservoir provide culture like baths
Armouries do not provide culture

MeuhV1 commented:
Fixed

(There was even another double "with", which is fixed of course)

dguenms committed:
Merge branch 'develop' into SmallFixes
